.github/PULL_REQUEST_TEMPLATE/default.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Pull Request
+---
+# What are the relevant tickets?
+<!--- If it fixes an open issue, please link to the issue here. -->
+<!--- Closes # --->
+<!--- Fixes # --->
+
+## Description (What does it do?)
+<!--- Describe your changes in detail -->
+
+## How can this be tested?
+<!--- Please describe in detail how your changes have been tested. -->
+<!--- Include details of your testing environment, any set-up required --->
+<!--- (e.g. data entry required for validation) and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+<!--- Please also include instructions for how your reviewer can validate your changes. --->
+
+
+## Screenshots (if appropriate):
+- [ ] Desktop screenshots
+- [ ] Mobile width screenshots
+
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] I have updated the documentation
+- [ ] My change includes new configuration settings that need to be updated in RC/Production
```

Comment:
Not sure who is this information for ?   What changes if I check or not check this box ?